docs/prod-host-setup.md

```diff
@@ -274,17 +274,38 @@ echo "swap partitions present (Recommendation: no swap)" \
 
 #### Side channel attacks
 
-It is strongly recommended that users follow the
+For the purposes of this document we assume a workload that involves arbitrary
+code execution in a multi-tenant context where each Firecracker process
+corresponds a single tenant.
+
+Specific mitigations for side channel issues are constantly evolving as
+researchers find additional issues (although, in general, of decreasing
+seriousness) on a regular basis. Firecracker itself has no control over many
```

Comment:
`although, in general, of decreasing seriousness` -> Is that really needed?

Reply:
No - removed